src/main/java/io/github/oliviercailloux/y2018/apartments/gui/AskOpinionForUtility.java

```diff
@@ -103,8 +99,8 @@ public static void main(String[] args) throws IllegalAccessException, IOExceptio
 
     LOGGER.info("Begining the Layout.");
 
-    LayoutApartmentGUI lay = new LayoutApartmentGUI(avf);
-    lay.displayAppart();
+    //    LayoutApartmentGUI lay = new LayoutApartmentGUI(avf);
+    //    lay.displayAppart();
```

Comment:
Suivez SVP BP Style

Reply:
fixed in b2175b5819224ac8d97e02a964906f3ece613ed5